msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
 
 **Optional** &ndash; Object
 
-Allows the specification of a default language and provides pointers to more language files. For more information, see [localization](~/concepts/build-and-test/apps-localization.md).
+Allows the specification of a default language, and pointers to additional language files. See [localization](~/concepts/build-and-test/apps-localization.md).
 
 |Name| Maximum size | Required | Description|
 |---|---|---|---|
-|`defaultLanguageTag`||✔️|The language tag of the strings in this top-level app manifest file.|
+|`defaultLanguageTag`||✔️|The language tag for the strings in this app manifest file. For example, `en`.|
```

Comment:
In the schema, seems the default value is `en-us`. Will that affect the new change? Do we need to check with the schema PM for this? or mention the default value in the doc?
![image](https://github.com/user-attachments/assets/2b2b28a3-3ff6-4c4f-a952-367a281dfb20)

Reply:
Ah, good catch! I've incorporated the default value to description.